Nos/Controller/SearchController.swift

```diff
@@ -101,6 +125,8 @@ class SearchController: ObservableObject {
     func clear() {
         query = ""
         authorResults = []
+        isNotFindingResults = false
```

Comment:
I think these two lines are redundant? Because mutating the query above will hit the combine pipeline and lines 84-85.

Reply:
You're absolutely right. Removing.

Reply:
Done!